pytest.python3.ini

```diff
@@ -1,2 +1,10 @@
 [pytest]
-addopts = --doctest-modules tests/python3/ augur/
+addopts =
+	# do not capture any output---necessary for interactive breakpoints
+	-s
+
+	# run any doctests found in .py modules
+	--doctest-modules
+
+	augur/
```

Comment:
Add these to `testpaths`, see here: https://github.com/pydata/sparse/blob/master/pytest.ini#L5-L6